### PR TITLE
refactor(ui5-radio-button): change tag name

### PR DESCRIPTION
**Pull request by gmkv**
Part of #3107

BREAKING_CHANGE: component's tag name `ui5-radiobutton` has been changed to `ui5-radio-button`

### CODE REVIEW

File: packages/fiori/src/NotificationListItem.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the elements, dipalyed in the footer of the of the <code>ui5-li-notification</code>.
+		 * Defines the elements, displayed in the footer of the of the <code>ui5-li-notification</code>.
```
Comment (fifoosid) [inline packages/fiori/src/NotificationListItem.js:98]: of the component

File: packages/main/src/RadioButton.js [hunk 2]
```diff
@@ -61,17 +62,17 @@ const metadata = {
...
-		 * either by cliking/tapping on the component,
+		 * either by clicking/tapping on the component,
 		 * or by using the Space or Enter key.
 		 *
 		 * @type {boolean}
 		 * @defaultvalue false
 		 * @public
 		 */
-		selected: {
+		checked: {
```
Comment (fifoosid) [inline packages/main/src/RadioButton.js:75]: please add a since tag

File: packages/main/src/RadioButton.js [hunk 3]
```diff
@@ -184,12 +185,12 @@ const metadata = {
...
 
 		/**
-		 * Fired when the component selected state changes.
+		 * Fired when the component checked state changes.
 		 *
 		 * @event
 		 * @public
 		 */
-		select: {},
+		change: {},
```
Comment (fifoosid) [inline packages/main/src/RadioButton.js:193]: please add a since tag

**Review by fifoosid — CHANGES_REQUESTED** [review]

**Review by ilhan007 — APPROVED** [review]

**Review by dimovpetar — CHANGES_REQUESTED** [review]
Also rename the CSS variables in RadioButton-parameters.css

**Comment by gmkv** [comment]
> Also rename the CSS variables in RadioButton-parameters.css

Done. Renamed all related vars (some were _ui5_rb_..., others - _ui5_radiobutton).

**Review by dimovpetar — APPROVED** [review]

**Pull request merged**